stdlib/crypto/src/main/java/org/ballerinalang/stdlib/crypto/CryptoUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.stdlib.crypto;
+
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.util.exceptions.BallerinaException;
+
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.Signature;
+import java.security.SignatureException;
+import javax.crypto.Mac;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * Utility functions relevant to crypto operations.
+ *
+ * @since 0.95.1
+ */
+public class CryptoUtils {
+
+    private CryptoUtils() {
```

Comment:
Do we need this private constructor?

Reply:
I was that this was a common practice in most of the Util classes found across stdlib, done to prevent someone from unintentionally initializing a util class with only bunch of static methods in it. Seems it's a good way to prevent misuse?